launch/tier4_simulator_launch/launch/simulator.launch.xml

```diff
@@ -97,10 +98,7 @@
   <!-- Dummy localization -->
   <group if="$(var launch_dummy_localization)">
     <include file="$(find-pkg-share pose_initializer)/launch/pose_initializer.launch.xml">
-      <arg name="gnss_enabled" value="false"/>
-      <arg name="ndt_enabled" value="false"/>
-      <arg name="ekf_enabled" value="false"/>
-      <arg name="stop_check_enabled" value="false"/>
+      <arg name="config_file" value="$(var pose_initializer_param_simulator_path)"/>
```

Comment:
In `autoware_launch/config/localization/pose_initializer.simulator.param.yaml`,     both of `ndt_enabled` and `ekf_enabled` are  true. 
However, neither EKF nor NDT will run in planning_simulation mode.

I guess this will NOT allow planning_simulator to complete the initial position estimation.

Reply:
@KYabuuchi Thank you! Resolved here: https://github.com/autowarefoundation/autoware.universe/pull/3038/commits/ad3e61b6eab4567ef64efe1f3813818311c6d806
Also confirmed that both planning simulator and logging simulator works

Reply:
Great!